src/libs/actions/PaymentMethods.ts

```diff
@@ -317,22 +332,24 @@ function clearWalletTermsError() {
     Onyx.merge(ONYXKEYS.WALLET_TERMS, {errors: null});
 }
 
-function deletePaymentCard(fundID) {
-    API.write(
-        'DeletePaymentCard',
-        {
-            fundID,
-        },
-        {
-            optimisticData: [
-                {
-                    onyxMethod: Onyx.METHOD.MERGE,
-                    key: `${ONYXKEYS.FUND_LIST}`,
-                    value: {[fundID]: {pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE}},
-                },
-            ],
-        },
-    );
+type DeletePaymentCardParams = {
```

Comment:
You can move params type to function body (examples in Fabio's [PR](https://github.com/Expensify/App/pull/28269))